src/plays/photoshopApp/Photoshopapp.js

```diff
@@ -0,0 +1,140 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import React, { useState } from 'react';
+// import './App.css';
+import Slider from './Slider';
+import SidebarItem from './SidebarItem';
+// WARNING: Do not change the entry componenet name
+
+const DEFAULT_OPTIONS = [
```

Comment:
Move this to constant.

Reply:
Sure Sir,Will Create a Seperate Constants file  and will import data from there

Reply:
Done and Committed Sir